### ISSUE TITLE
[OS X] Окно не в фокусе после запуска

### ISSUE THREAD

**Issue by yunoshev** [ReanGD/LearningEnglish#33]
После выполнение в терминале python ./main.py  - окно появляется, под терминалом

P.S. Напишите - имеет смысл постить замечание по работе под osx - а то их много.

**Comment by ReanGD** [comment]
Да, пишите конечно, если не трудно. Я постараюсь в ближайшее время отыскать дистрибутив OS X, установить на виртуалку и поправить баги. Заодно прокачаю скилл работы с не Windows операционными системами )
